apps/token/src/routes/governance/components/current-proposal-status/current-proposal-status.tsx

```diff
@@ -34,37 +34,53 @@ export const CurrentProposalStatus = ({
     { addSuffix: true }
   );
 
-  if (proposal.state === ProposalState.Open && willPass) {
-    return <StatusPass>{t('shouldPass')}</StatusPass>;
-  }
-
-  if (!participationMet) {
-    return (
-      <>
-        <span>{t('voteFailedReason')}</span>
-        <span className="current-proposal-status__fail">
-          {t('participationNotMet')}
-        </span>
-        <span>&nbsp;{daysClosedAgo}.</span>
-      </>
-    );
-  }
-
-  if (!majorityMet) {
-    return (
-      <>
-        <span>{t('voteFailedReason')}</span>
-        <StatusFail>{t('majorityNotMet')}</StatusFail>
-        <span>&nbsp;{daysClosedAgo}.</span>
-      </>
-    );
+  if (proposal.state === ProposalState.Open) {
+    if (willPass) {
+      return (
+        <>
+          Currently set to{' '}
```

Comment:
Add to transaltion file
```suggestion
         t('Currently set to')
```